app/main.py

```diff
@@ -265,32 +266,45 @@ def main():
     )
 
     sleep_count = 0
-    state = identity_record.attestation_details.state
+    account_state = account.state
+    final_states = [STATE_CREATED]
+    while account_state not in final_states and sleep_count < Config.TIMEOUT:
+        time.sleep(1)
+        sleep_count += 1
+        account = get_account(api_client, account.guid)
+        account_state = account.state
+    if account_state != STATE_CREATED:
```

Comment:
Shouldn't this similarly be `if account_state not in final_states`?

Reply:
No; `created` is the only "good" state, just like identity records below. Now, we only have one final state, and it lines up with the "good" one, but that could change at some point.